scripts/scil_image_math.py

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+"""
+Performs an operation on a list of images. The supported operations are
+listed below.
+
+This script is loading all images in memory, will often crash after a few
+hundred images.
```

Comment:
Maybe we should detect this with a heuristic or send a warning if 100+ images given?

Reply:
This should be in another PR, maybe make an issue?
But nowhere in Scilpy we handle RAM or warn people about that.
Screenshotting bundle loads everything in RAM too (my similarity script too).